homeassistant/components/icloud/account.py

```diff
@@ -426,6 +436,18 @@ def update(self, status) -> None:
                 and self._status[DEVICE_LOCATION][DEVICE_LOCATION_LATITUDE]
             ):
                 location = self._status[DEVICE_LOCATION]
+                if (
+                    self._config_entry.data[CONF_GPS_ACCURACY_THRESHOLD] is not None
```

Comment:
No worries! Funnily enough, I learned about protected variables through a Home Assistant contribution code review. Go figure!

Reply:
I just changed it. Created a Getter-method in `IcloudAccount` instead of making the variable public. Honestly can't imagine why I didn't think of this before...